.vortex/installer/src/Utils/FileManager.php

```
@@ -87,9 +114,76 @@ public function copyFiles(): void {
+    $dirs = [];
+
+    foreach ($paths as $path) {
+      // The harness never ships, so a matching path in the destination is the
+      // project's own.
+      if ($path === self::HARNESS_DIR || str_starts_with($path, self::HARNESS_DIR . '/')) {
+        continue;
+      }
+
+      $target = $destination . '/' . $path;
+
+      if (File::exists($target)) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'DrevOps\VortexInstaller\Utils\File' in method 'removeExcludedPaths'. (undefined)

(StaticAccess)